publication-model/src/main/java/no/unit/nva/model/associatedartifacts/file/PendingInternalFile.java

```diff
@@ -85,4 +87,9 @@ public InternalFile approve() {
                                 getPublisherVersion(), getEmbargoDate().orElse(null), getRightsRetentionStrategy(),
                                 getLegalNote(), Instant.now(), getUploadDetails());
     }
+
+    @Override
+    public boolean isNotApprovable() {
```

Comment:
We really need to document what the different files represent inline in the code.

Reply:
You think something like JavaDoc for each single file class? or maybe attach link to confluence?

Reply:
Yes, javadoc. New PR